1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,41 @@
-
+import fizzbuzz from './fizzbuzz';
 describe("fizzbuzz", () => {
 
+  it("return error if number is less than 1", () => {
+    expect(() => fizzbuzz(0)).toThrow();
+  });
+
+  it("return error if number is more than 100", () => {
+    expect(() => fizzbuzz(101)).toThrow();
+  });
+
+  it("returns a string", () => {
+    expect(typeof fizzbuzz(5)).toBe('string');
+  });
+
+  it("returns the number as a string", () => {
+    expect(fizzbuzz(1)).toBe('1');
```

Comment:
Similar to the multiples of 3 and 5 below, could this also be a more exhaustive test?

Reply:
Agree with that.